src/i18n/_locales/ja/messages.json

```diff
@@ -1,10 +1,113 @@
 {
+  "gitHasLocalChanges": {
+    "message": "with local changes" 
+  },
```

Comment:
Can this string be translated too?
You can see it in the "about" dialog bog (red text):
https://readium.firebaseapp.com
The message indicates that the local Git repository wasn't clean when the app was built.